net-libs/gssdp/gssdp-1.6.2.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson-multilib vala xdg
+
+DESCRIPTION="GObject-based API for handling resource discovery and announcement over SSDP"
+HOMEPAGE="https://wiki.gnome.org/Projects/GUPnP https://gitlab.gnome.org/GNOME/gssdp"
+
+LICENSE="LGPL-2+"
+SLOT="1.6/1.6-0" # <API version>-<soname>
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="gtk gtk-doc +introspection +vala manpages"
+REQUIRED_USE="
+	gtk-doc? ( introspection )
+	vala? ( introspection )
+"
+
+# has file collision with Slot 0 (/usr/bin/gssdp-device-sniffer), so we remove that from this build
+# and depend on the other slot that has the file as well
+RDEPEND="
+	>=dev-libs/glib-2.70:2[${MULTILIB_USEDEP}]
+	>=net-libs/libsoup-2.99.0:3.0[${MULTILIB_USEDEP},introspection?]
+	gtk? ( gui-libs/gtk:4 )
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
+	manpages? ( app-text/pandoc )
```

Comment:
I think this should be `virtual/pandoc` instead.

Reply:
Also, pandoc is not stable, so in preparation for eventually stabilizing gssdp we should add an entry to `base/package.use.stable.mask` masking this new man page flag except on unstable.

Reply:
Solved in https://github.com/gentoo/gentoo/commit/d9387b41347e672eff6777cbff5ce078e0e58619